lib/src/commands.rs

```diff
@@ -78,18 +78,18 @@ impl Command for ParseCommand {
         match args.len() {
             1 => json::stringify_pretty(
                 [
-                    zingoconfig::BlockChain::Mainnet,
-                    zingoconfig::BlockChain::Testnet,
-                    zingoconfig::BlockChain::Regtest,
+                    zingoconfig::ChainType::Mainnet,
+                    zingoconfig::ChainType::Testnet,
+                    zingoconfig::ChainType::Regtest,
                 ]
                 .iter()
                 .find_map(|chain| RecipientAddress::decode(chain, &args[0]).zip(Some(chain)))
                 .map(|(recipient_address, chain_name)| {
                     let chain_name_string = match chain_name {
```

Comment:
I don't know much of rust, but I think here is still the field named `chain_name`

Reply:
Well...  it's a local variable.   Do you prefer another name?  I thought we wanted to name it `chain_name`?

Reply:
If this a local var... sorry. My bad...